resources/[system]/chat/html/Message.js

```diff
@@ -0,0 +1,65 @@
+Vue.component('message', {
+  template: '#message_template',
+  data() {
+    return {};
+  },
+  computed: {
+    textEscaped() {
+      let s = '';
+      if (this.template) {
+        s = this.colorize(this.template);
+      } else {
+        s = this.colorize(this.templates[this.templateId]);
+      }
+      return s.replace(/{(\d+)}/g, (match, number) => {
+        const argEscaped = this.args[number] != undefined ? this.escape(this.args[number]) : match
+        if (number == 0 && this.color) {
+          //color is deprecated, use templates or ^1 etc.
```

Comment:
the old `color` behavior was to colorize merely the name, yet stick with ^1/... for the message itself. is this still done here?